src/Symfony/Bridge/Twig/Extension/ImportMapsExtension.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\ImportMaps\ImportMapManager;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+final class ImportMapsExtension extends AbstractExtension
+{
+    public function __construct(
+        private readonly ImportMapManager $importMapManager,
+    )
+    {
+    }
+
+    public function getFunctions(): iterable
+    {
+        yield new TwigFunction('importmap', [$this, 'importmap'], ['is_safe' => ['html']]);
+    }
+
+    public function importmap(bool $polyfill = true): string
```

Comment:
Is it OK to have a unique importmap for all the application even if it use different JS modules on different pages?

Using several importmaps on a single page is documented: https://github.com/WICG/import-maps#multiple-import-map-support

As in the Asset component, we need to configure multiple importmaps and provide the name to the Twig helper.

Reply:
It’s supported. You’ll have to register an extra ImportMap manager. Maybe could we automate that trough config at some point, but for now I don’t think it’s necessary. Rails uses only one import map for instance.

Reply:
We can register an other `ImportMapManager` service, but the twig helper has a unique name.

Reply:
In the future, we could path an optional name to the helper?